app/actions/client.py

```diff
@@ -0,0 +1,104 @@
+import logging
+import httpx
+import pydantic
+import stamina
+
+from typing import Optional
+from app.services.state import IntegrationStateManager
+
+
+logger = logging.getLogger(__name__)
+state_manager = IntegrationStateManager()
+
+
+class FarmResponse(pydantic.BaseModel):
+    id: str = pydantic.Field(alias='_id')
+    name: str
+    nif: Optional[str]
+    rega: Optional[str]
+
+    class Config:
+        allow_population_by_field_name = True
+
+
+class FarmLocationResponse(pydantic.BaseModel):
+    location: str = pydantic.Field(alias='_location')
+    time: str = pydantic.Field(alias='_time')
+    device_name: str
+    official_tag: str
+
+    class Config:
+        allow_population_by_field_name = True
```

Comment:
Is this the model for a single location?  You might want to give this a different name other than _Response_ so it won't be confused for a response body.